lightning/src/ln/peer_handler.rs

```diff
@@ -1088,6 +1094,43 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 			}
 		};
 	}
+
+	/// This function can be called at an interval greater than every 30 seconds, it will:
```

Comment:
There is a bit too much internal guts described in this comment. Maybe something a bit simpler (and more prescriptive), eg:
```
This function should be called roughly once every 30 seconds.
It will send pings to each peer and disconnect those which did not respond to the last round of pings.

Will most likely call send_data on all of the registered descriptors, thus, be very careful with reentrancy issues!
```